lib/audit/checksum.rb

```diff
@@ -62,6 +62,8 @@ def self.validate_druid(druid)
     end_msg = "#{Time.now.utc.iso8601} CV validate_druid ended for #{druid}"
     puts end_msg
     Rails.logger.info end_msg
+  rescue NoMethodError => e
```

Comment:
if we're taking this approach (assuming there's one entry in `preserved_copies` per druid, which is safe as long as we haven't gotten to archive copies yet), we should do a `find_by!` instead of a `find_by`, and then we get a nice active record error about the object not being found, instead of getting a `nil` back and inferring from the `NoMethodError`.

Reply:
See #263 re: one pres_copy re druid-endpoint

Reply:
i'm totally fine with assuming one PC per druid for now since that's the way things currently are, and then revisiting under #263.  and i'd approve and merge this as-is if people thought the above comment was too nitpicky.  but using `find_by!` instead of `find_by` is more consistent with our other code (and is a thing we deliberately standardized on in POH), and i think `find_by!` would provide a more informative exception, so i figured it'd be worth proposing that if we did stick with the `find_by` approach in general.

Reply:
(and then if we wanted to trap and log something, we could specifically trap `ActiveRecord::RecordNotFound`)